apps/rsa.c

```diff
@@ -223,7 +223,7 @@ int rsa_main(int argc, char **argv)
 
     if (!opt_cipher(ciphername, &enc))
         goto opthelp;
-    private = (text && !pubin) || (!pubout && !noout) ? 1 : 0;
+    private = (text && !pubin) || (!pubout && !noout);
```

Comment:
See above.

Reply:
The ` ? 1 : 0` is just needless and potentially confusing.
I really think it's better just dropping it.